libraries/GCS_MAVLink/GCS.h

```diff
@@ -759,6 +759,7 @@ class GCS_MAVLINK
         uint32_t remrssi_ms;
         uint8_t rssi;
         uint32_t received_ms; // time RADIO_STATUS received
+        uint8_t txbuf = 100;
```

Comment:
With the new timeout we don't need to init to 100 anymore. 
```suggestion
        uint8_t txbuf;
```

Reply:
If we add the check for 0 ms.

Reply:
It seems more efficient to initialize just once than to execute the extra check every time.